scipy/stats/tests/test_resampling.py

```diff
@@ -653,6 +650,44 @@ def statistic(data):
     assert res.bootstrap_distribution.shape == (2, 100, 9999)
 
 
+@pytest.mark.slow
+@pytest.mark.filterwarnings('ignore::RuntimeWarning')
+def test_vector_valued_statistic_gh17715():
+    # gh-17715 reported a mistake introduced in the extension of BCa to
+    # multi-sample statistics; a `len` should have been `.shape[-1]`. Check
+    # that this is resolved.
+
+    rng = np.random.default_rng(141921000979291141)
+
+    def concordance(x, y, axis):
+        xm = x.mean(axis)
+        ym = y.mean(axis)
+        cov = ((x - xm[..., None]) * (y - ym[..., None])).mean(axis)
+        return (2 * cov) / (x.var(axis) + y.var(axis) + (xm - ym) ** 2)
+
+    def statistic(tp, tn, fp, fn, axis):
+        actual = tp + fp
+        expected = tp + fn
+        return np.nan_to_num(concordance(actual, expected, axis))
```

Comment:
This is essentially the same as the code in the original issue gh-17715, but vectorized so that this test is fast enough to run regularly. Can you confirm that this test does not pass in 1.10 and is essentially the same as what you had, @slanzmich?